.github/ISSUE_TEMPLATE/update_truth.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Update Truth
+about: Review use case differences that are caused by changes in an external repository and update truth dataset if necessary.
+title: 'Update Truth: '
+labels: 'alert: NEED ACCOUNT KEY, alert: NEED MORE DEFINITION, alert: NEED CYCLE ASSIGNMENT, type: update truth, priority: blocker, component: CI/CD'
+assignees: ''
+
+---
+
+## Describe Expected Changes ##
+
+*Write a short summary of the differences that will likely be found in the GitHub Actions testing workflow that was triggered by the pull request that warranted this issue*
+
+## Define the Metadata ##
+
+### Title ###
+- [ ] Add a link to the pull request that warranted this issue to the issue title using format dtcenter/{REPO}#{PR_NUMBER}.
+
+**Example:** Update Truth: For dtcenter/MET#2655
+
+### Assignee ###
+
+*Assign this issue to the author of the pull request that warranted this issue. Optionally assign anyone else who should review the differences in the output.*
+
+- [ ] Select **engineer(s)** or **no engineer** required
+- [ ] Select **scientist(s)** or **no scientist** required
+
+### Labels ###
```

Comment:
Can we remove the "Labels" section since they're already added by default?

Reply:
Yes, good point.